Docs: Add source code links to OpenVINO Samples (#11803)
openvinotoolkit/openvino: samples/python/speech_sample/README.md

# Automatic Speech Recognition Python* Sample {#openvino_inference_engine_ie_bridges_python_sample_speech_sample_README}

> This sample demonstrates how to do a Synchronous Inference of acoustic model based on Kaldi\* neural models and speech feature vectors. The source code for this example is also available [on GitHub](https://github.com/openvinotoolkit/openvino/tree/master/samples/python/speech_sample).

Reviewer: ```suggestion
This sample demonstrates how to execute asynchronous inference of an acoustic model based on Kaldi neural models and speech feature vectors. Source code for this example is also available [on GitHub](https://github.com/openvinotoolkit/openvino/tree/master/samples/python/speech_sample).
```
